### PR TITLE
Implementing controller and adding methods to service

### PR DESCRIPTION
**Pull request by VictorBarronG**

### CODE REVIEW

**Review by kpugh97 — APPROVED** [review]
I would leave a TODO over your methods with placeholders just so you remember to go back to them

**Comment by VictorBarronG** [comment]
> I would leave a TODO over your methods with placeholders just so you remember to go back to them

I gotcha ill do that

File: findmyparadise/src/main/java/com/tp/wrc/findmyparadise/controllers/PhotoGalleryController.java [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 import com.tp.wrc.findmyparadise.services.PhotoGalleryService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("api")
```
Comment (jwmann13) [inline findmyparadise/src/main/java/com/tp/wrc/findmyparadise/controllers/PhotoGalleryController.java:10]: You should probably make the overall request mapping more specific to this controller.
Right now all the endpoint are just on `"/api"` e.g. `"/api/getById/{id}"` which will likely cause failures and collisions with the other controllers.
Try setting `@RequestMapping("api/photos")`or something like that

Comment (VictorBarronG) [reply findmyparadise/src/main/java/com/tp/wrc/findmyparadise/controllers/PhotoGalleryController.java:10]: I gotcha ill make those changes too

**Pull request merged**